CSharp/BatchExplorer/ViewModels/ResizePoolViewModel.cs

```diff
@@ -72,6 +86,34 @@ public string DeallocationOptionString
                 this.deallocationOptionString = value;
                 this.FirePropertyChangedEvent("DeallocationOptionString");
             }
+        }
+
+        private bool isFixed;

```

Comment:
You are absolutely right. This can be achieved with a single property as well. Made the required changes.